src/main.cpp

```
@@ -5,31 +5,39 @@
+    QApplication app(argc, argv);
+    DeviceManager manager;
 
-  MainWindow window(&manager);
+    auto *zigbee = new ZigbeeProvider(manager.mqtt_client(), // NOLINT
+                                      &manager);             // NOLINT
+    manager.register_provider(zigbee);
 
-  window.showFullScreen();
+    QFile file(style_path);
+    if (file.open(QFile::ReadOnly | QFile::Text)) {
+      QTextStream stream(&file);
```

Clang (14.0.6)

[warning] variable 'stream' is not initialized

(cppcoreguidelines-init-variables)